lib/pattern_query_helper/associations.rb

```diff
@@ -10,8 +10,9 @@ def self.process_association_params(associations)
     end
 
     def self.load_associations(payload, associations, as_json_options)
+      as_json_options ||= {}
+      as_json_options[:include] = Array(as_json_options[:include]) + json_associations(associations)
```

Comment:
My only thought would be if we should update this to just use one or the other instead of trying to combine them the two if both are passed: 

```suggestion
      as_json_options[:include] = as_json_options[:include] || json_associations(associations)
```

I'm not sure when we would pass includes for the associations and the as_json options.